game.js

```diff
@@ -9,146 +9,369 @@ Here, we create and add our "canvas" to the page.
 We also load all of our images. 
 */
 
-
 let canvas;
 let ctx;
 
-canvas = document.createElement("canvas");
+canvas = document.querySelector("canvas");
 ctx = canvas.getContext("2d");
 canvas.width = 512;
-canvas.height = 480;
-document.body.appendChild(canvas);
+canvas.height = 488;
+
+// define max Egg collected and max duration
+let maxEgg = 12;
+let maxTime = 30;
+
+// document.body.appendChild(canvas);
+
+/* declare global variables*/
+
+/*declare 3 objects*/
+let bgReady, dinoReady, eggReady;
+let bgImage, dinoImage, eggImage;
 
-let bgReady, heroReady, monsterReady;
-let bgImage, heroImage, monsterImage;
+/* set-up function, button reset game*/
+let newGameBtn = document.getElementById("resetBtn");
+newGameBtn.addEventListener("click", Reset);
+
+/*declare variables for remaining time*/
+let secondsSpan = document.getElementById("remain-time");
+var timeinterval;

```

Comment:
Just FYI, could be good to get into the habit of using "let" instead of "var" to declare variables, because i've heard "let" is more common on modern frameworks